static/js/tools/map/chart_loader.tsx

```diff
@@ -339,10 +340,11 @@ function fetchData(
       return resp.data;
     });
 
+  const mapPointSv = statVar.mapPointSv || statVar.dcid;
   const mapPointDataPromise: Promise<GetStatSetResponse> = placeInfo.mapPointPlaceType
     ? axios
         .get(
-          `/api/stats/within-place?parent_place=${placeInfo.enclosingPlace.dcid}&child_type=${placeInfo.mapPointPlaceType}&stat_vars=${statVar.dcid}${dateParam}`
+          `/api/stats/within-place?parent_place=${placeInfo.enclosingPlace.dcid}&child_type=${placeInfo.mapPointPlaceType}&stat_vars=${mapPointSv}`
```

Comment:
should we keep the dateParam?

Reply:
keeping dateParam doesn't work well when the stat var for the map is different for the installations .. for example, RCP statvars will use 2050, but not many other stat vars will have data for 2050

Reply:
that makes sense. could you update the pr description with the update? it's good to list all these changes out, which helps with future debugging, and/or a comment in code.

Reply:
done, updated pr description